File: WebApi/Web_API/DataLayer/Models/SensorValue.cs
using System.ComponentModel.DataAnnotations;

namespace DataLayer.Models
{
    public class SensorValue
    {
        [Key]
        public int SensorValueId { get; set; } // primary key

        [Required]
        public double Value { get; set; }

        [Required]
        public int SensorId { get; set; } // foreign key

        [Required]
        public int SessionId { get; set; } // foreign key

        [Required]
        public int PackageId { get; set; }

        public virtual Sensor Sensor { get; set; } // required for foreign key
        public virtual Session Session { get; set; } // required for foreign key
    }
}

File: DataSender/scripts/sensor_value.py
class SensorValue:
  def __init__(self, value, sensor_id):
    self.value = value
    self.sensor_id = sensor_id
    self.session_id  = 0

  def repr_JSON(self):
    return dict(value=self.value, sensor_id=self.sensor_id, session_id=self.session_id) 